reactioncommerce/reaction
# fix blank shipping data cancel order

Resolve #3075 

This PR try to display a message when an order is cancelled or removed. A blank area is shown in the shipping section whenever an order is cancelled. 

#### Test
* Loggin as Admin.
* Place an order.
* Checkout  and complete order.
* Click on order icon at the right.
* On the order row, click on the blue box with a white arrow.
* The order summary will slide out.
* Scroll to the `invoice` section.
* Toggle the `Approve` button to reveal `Cancel`.
* Click on the `Cancel` button.
* Scroll down to shipping section to see the `shipping info` displayed.
